config/example/config.toml

```diff
@@ -295,7 +295,7 @@ server_name = "ParaCode Test"
 # EG: The perfect mix of RP & action
 server_tag_line = "ParaCode Testing Server"
 # Server features. Shows in a newline
-server_features = "Medium RP, varied species/jobs."
```

Comment:
Will need to update on prod

Reply:
It's actually "MedRP, varied species and jobs." on prod currently.